tests/envs/test_cover.py

```diff
@@ -117,6 +117,81 @@ def test_cover():
     assert "did not implement an interface for human demonstrations" in str(e)
 
 
+def test_cover_hand_empty():
```

Comment:
a lot of this looks the same as the regular cover test. let's use parameterized testing to remove the redundant code. then if there are specific things about the CoverEnvHandEmpty environment that we should check, we can just include those things in this function.

to see what I mean by "parameterized test", here are a few examples:
* https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/tests/explorers/test_glib_explorer.py#L12
* https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/tests/approaches/test_gnn_option_policy_approach.py#L46
* https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/tests/pybullet_helpers/test_link.py#L31